.github/workflows/main.yml

```
@@ -46,21 +46,23 @@ jobs:
         id: setup
         with:
           ghc-version: ${{ matrix.ghc }}
-          cabal-version: 'latest'
+          cabal-version: "latest"
           cabal-update: true
       - name: Configure the build
         run: |
-          cabal configure --enable-tests --disable-documentation
+          cabal configure --enable-tests --enable-benchmarks --disable-documentation
           cabal build all --dry-run
+
       - name: Restore cached dependencies
         uses: actions/cache/restore@v4
         id: cache
         env:
           key: ${{ runner.os }}-ghc-${{ steps.setup.outputs.ghc-version }}-cabal-${{ steps.setup.outputs.cabal-version }}
         with:
           path: ${{ steps.setup.outputs.cabal-store }}
-          key: ${{ env.key }}-plan-${{ hashFiles('**/plan.json') }}
-          restore-keys: ${{ env.key }}-
+          key: v1-ghc-deps-${{ env.key }}-plan-${{ hashFiles('**/dist-newstyle/cache/plan.json') }}
```

@davidnbr I think the addition of a version would be useful; however, I believe we could shorten the cache key a bit because there is only one cache used in the entire build:

```suggestion
          key: v1-${{ env.key }}-plan-${{ hashFiles('**/dist-newstyle/cache/plan.json') }}
```

---

Thanks for this feedback @sestrella ! I addressed it.